Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'slop', '~> 4.0', platforms: [:ruby_20, :ruby_21, :ruby_22]
```

Comment:
I'll try it to satisfy my curiosity! :smile:

I'm currently very busy but I'll try to migrate to optparse when I get time if nobody does it before. I recently migrated byebug and it was pretty painless.